app/views/shared/_sentry.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if ENV['SENTRY_DSN'].present? %>
```

Comment:
Do we need sentry in the Frontend? Or does it catch any unhandled exceptions?

Reply:
yes exactly. i would at least use it before the camp starts. let's see if we are overwhelmed with FE errors from weird browsers etc.